e2e/tests/dropdownMoveFocusOnTab-example.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react'
+import { Dropdown } from '@stardust-ui/react'
+
+const inputItems = [
+  'Bruce Wayne',
+  'Natasha Romanoff',
+  'Steven Strange',
+  'Alfred Pennyworth',
+  `Scarlett O'Hara`,
+  'Imperator Furiosa',
+  'Bruce Banner',
+  'Peter Parker',
+  'Selina Kyle',
+]
+
+const DropdownExampleMultiple = () => (
+  <>
+    <button id="previous-button" />
```

Comment:
names like `previous-focusable-sibling` and `next-focusable-sibling` will help a lot to understand the test's intent (as, in fact, we don't care if it is a button or anything else, only `focusable` trait matters to us). 

As an example, this is how test's code will look like with these names:

```jsx
 await e2e.pressKey('ArrowDown') // open dropdown list
 await e2e.pressKey('Tab') // TAB from opened dropdown list

 expect(await e2e.isFocused(triggerButton)).toBe(false)

 expect(await e2e.isFocused(nextFocusableSibling)).toBe(true)
// vs  expect(await e2e.isFocused(nextButton)).toBe(true)
```